reledmac.dtx

```diff
@@ -312,20 +312,19 @@
 % \changes{v2.0.0}{2015/07/19}{Fix bug when printing only one series of endnotes, but wanted to keep endnotes for other series.}
 % \changes{v2.0.1}{2015/07/27}{Fix incompatibility between optional argument of \protect\cs{pstart} and \protect\cs{numberpstarttrue}}
 % \changes{v2.0.1}{2015/07/27}{Fix bug in eledmac-compat option}
-% \changes{v2.1.0}{2015/08/08}{New tools to number stanza}
-%
-%\changes{v2.1.0}{2015/08/02}{Fix bug with \protect\cs{chapter} in optional argument of \protect\cs{pstart} in parallel typesetting with \protect\ltxclass{scrbook}.}
+% \changes{v2.1.0}{2015/08/08}{New tools to number stanzas}
+% \changes{v2.1.0}{2015/08/02}{Fix bug with \protect\cs{chapter} in optional argument of \protect\cs{pstart} in parallel typesetting with \protect\ltxclass{scrbook}.}
 % \changes{v2.1.0}{2015/08/02}{Fix bug with \protect\cs{eledchapter} in parallel typesetting with \protect\ltxclass{scrbook}.}
-% \changes{v2.1.0}{2015/08/02}{Fix bug with \protect\cs{setline} at the begin of a \protect\cs{pstart}.}
-% \changes{v2.1.0}{2015/08/02}{Fix bug with \protect\cs{advanceline} at the begin of a \protect\cs{pstart}.}
-% \changes{v2.1.0}{2015/08/08}{Fix space bug with \protect\cs{Xbhooknote} and \protect\cs{bhooknoteX} when using to insert text and not to execute code.}
-% \changes{v2.1.0}{2015/08/08}{Fix space bug with \protect\cs{Xbhooknote} and \protect\cs{bhooknoteX} when using to insert text and not to execute code.}
+% \changes{v2.1.0}{2015/08/02}{Fix bug with \protect\cs{setline} at the beginning of a \protect\cs{pstart}.}
+% \changes{v2.1.0}{2015/08/02}{Fix bug with \protect\cs{advanceline} at the beginning of a \protect\cs{pstart}.}
+% \changes{v2.1.0}{2015/08/08}{Fix spacing bug with \protect\cs{Xbhooknote} and \protect\cs{bhooknoteX} when using them to insert text and not to execute code.}
+% \changes{v2.1.0}{2015/08/08}{Fix spacing bug with \protect\cs{Xbhooknote} and \protect\cs{bhooknoteX} when using them to insert text and not to execute code.}
 % \changes{v2.1.1}{2015/08/13}{Fix bug with \protect\cs{ledpbsetting}\protect\arg{before}.}
-% \changes{v2.1.2}{2015/08/26}{Fix bug with lineation by pstart and tabular environment (added in 2.1.0).}
-% \changes{v2.1.3}{2015/09/01}{Fix bug with \protect\cs{edindex} when using not-latin characters with not utf8 engines}
+% \changes{v2.1.2}{2015/08/26}{Fix bug with lineation with pstart and tabular environments (added in 2.1.0).}
```

Comment:
I changed the preposition from “by” to “with”, since in idiomatic American English, one changes something “with” something else. For example, “I changed the battery with my tools”.


On Sep 4, 2015, at 16:22 , Maïeul Rouquette <notifications@github.com> wrote:

> In reledmac.dtx:
> 
> >  % \changes{v2.1.1}{2015/08/13}{Fix bug with \protect\cs{ledpbsetting}\protect\arg{before}.}
> > -% \changes{v2.1.2}{2015/08/26}{Fix bug with lineation by pstart and tabular environment (added in 2.1.0).}
> > -% \changes{v2.1.3}{2015/09/01}{Fix bug with \protect\cs{edindex} when using not-latin characters with not utf8 engines}
> > +% \changes{v2.1.2}{2015/08/26}{Fix bug with lineation with pstart and tabular environments (added in 2.1.0).}
> why with ? lineation restart at each pstart
> 
> —
> Reply to this email directly or view it on GitHub.
>